aten/src/ATen/native/Collectives.cpp

```diff
@@ -24,6 +24,10 @@ at::Tensor all_gather_into_tensor(at::Tensor const& shard, const c10::string_vie
     TORCH_INTERNAL_ASSERT(false);
 }
 
+at::Tensor reduce_scatter_tensor(at::Tensor const& shard, const c10::string_view reduceOp, int64_t scatter_dim, const c10::string_view tag, c10::ArrayRef<int64_t> ranks, int64_t group_size) {
```

Comment:
is the naming correct?  shard is the input to reduce_scatter? or shard is the output and ... idk, source, tensor, etc is the input?

Reply:
Sorry was a copy-pasta issue haha, I think maybe we can just use "input" to align with the eager API naming? Updated